docs/conf.py

```diff
@@ -28,7 +28,7 @@
 autodoc_mock_imports = ["busio", "adafruit_register", "adafruit_bus_device"]
 
 autodoc_preserve_defaults = True
-
+# autodoc_member_order = "bysource"
 
```

Comment:
I experimented with this docs configuration. I noticed by default all of the attributes and properties are listed in alphabetical order which is nice in some ways. But it means that things like low threshold and high threshold pairs don't end up being next to each other in the docs page in some cases. 

This setting makes it use the order they appear in the code instead of alphabetical. I did turn it back off for now though because I wasn't sure if we'd want to use it, it's not set this way in any other library, so I'm hesitant. 

What do you think about renaming `als_high_threshold` -> `als_threshold_high`  and `als_low_threshold` -> `als_threshold_low` that way they'll match the order of the other threshold values and they'll end up being pals next to each other on the docs page.

Reply:
i think renaming them makes sense. if no other library uses "bysource" then we should probably leave it alphabetical. i think that with the prox/als prefixes and the examples folks should be able to sort things out